cartoframes/utils/utils.py

```diff
@@ -299,3 +299,12 @@ def get_center(center):
         return None
 
     return [center.get('lng'), center.get('lat')]
+
+
+def remove_column_from_dataframe(dataframe, name):
+    """Removes a column or index (or both) from a DataFrames"""
+    if name in dataframe.columns:
+        del dataframe[name]
+    if dataframe.index.name == name:
+        dataframe.reset_index(inplace=True)
+        del dataframe[name]
```

Comment:
I think this is going to raise `KeyError` because the column has been deleted 4 lines above.

Reply:
It works because `dataframe.reset_index(inplace=True)` creates a new index and moves the existing one to a column. This is what I've observed at least, maybe we should protect the second `del` with another `if name in dataframe.columns` in case that index-to-column doesn't happen in all cases (I admit I have no clue about how those dataframes work)

Reply:
:+1: